pr_agent/algo/utils.py

```diff
@@ -61,3 +64,25 @@ def parse_code_suggestion(code_suggestions: dict) -> str:
     markdown_text += "\n"
     return markdown_text
 
+
+def try_fix_json(review, max_iter=10):
+    # Try to fix JSON if it is broken/incomplete: parse until the last valid code suggestion
```

Comment:
Consider using a for loop instead of a while loop to iterate over the maximum number of iterations. This would make the code more readable and eliminate the need for manually incrementing the iter_count variable. [important]